package.json

```diff
@@ -64,19 +65,20 @@
     "jest": "^23.6.0",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.5.0",
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
     "react-ga": "^2.5.6",
-    "react-styleguidist": "^8.0.6",
+    "react-styleguidist": "9.0.4",
+    "react-test-renderer": "^16.9.0",
```

Comment:
Upgraded some of these libraries to be compatible with React Hooks:
https://github.com/airbnb/enzyme/issues/1938